requirements/index.html

```diff
@@ -42,7 +42,7 @@ <h2>Introduction</h2>
     <h2>Use Cases</h2>
     <section id="smart-home-use-cases">
       <h3>Smart Home</h3>
-      <p>As a developer of DIY smart home hub software, I want to safely expose a standardised HTTP-based API for
+      <p>As a developer of DIY smart home hub software, I want to safely expose a standardised web-based API for
```

Comment:
Since we're operating within the W3C umbrella, we should use US spelling.
```suggestion
      <p>As a developer of DIY smart home hub software, I want to safely expose a standardized web-based API for
```